app/src/main/res/layout-sw600dp/administrator_controls_app_information_view.xml

```diff
@@ -22,14 +22,13 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
```

Comment:
`android:layout_height="wrap_content"` can be removed as it is already defined in `Caption -> TextView.Common1`